include/boost/spirit/home/x3/support/traits/print_attribute.hpp

```diff
@@ -14,6 +14,7 @@
 #include <boost/fusion/include/for_each.hpp>
 #include <boost/spirit/home/x3/support/traits/attribute_category.hpp>
 #include <boost/spirit/home/x3/support/traits/is_variant.hpp>
+#include <boost/spirit/home/support/char_encoding/unicode.hpp>
```

Comment:
I blazed through changes for the release and stuck on this line. It unconditionally includes the header which we previously only included when `BOOST_SPIRIT_X3_UNICODE` is defined.

Reply:
Oh. That slipped through. We'll need to fix this for the 1.76.0 Beta candidate. @theodelrieu ?

Reply:
I already fixed it in develop. Beta is already out, and master branch is closed, but we will have an opportunity to merge it for the release later on.